web/src/recent_view_ui.js

```diff
@@ -1390,4 +1468,17 @@ export function initialize({on_click_participant, on_mark_pm_as_read, on_mark_to
         $("#recent_view_search").val("");
         update_filters_view();
     });
+
+    $("body").on("click", ".recent-view-load-more-container .fetch-messages-button", () => {
+        message_fetch.maybe_load_older_messages({
+            msg_list: message_lists.home,
+            recent_view: true,
+        });
+        $(".recent-view-load-more-container .button-label").toggleClass("invisible", true);
+        $(".recent-view-load-more-container .fetch-messages-button").prop("disabled", true);
+        loading.make_indicator(
+            $(".recent-view-load-more-container .fetch-messages-button .loading-indicator"),
+            {width: 20},
+        );
+    });
```

Comment:
You can fix the import cycle by passing `message_fetch.maybe_load_older_messages` into `initialize` like the handful of other methods are already.

Reply:
did that!